joomla/joomla-cms
# [4.0] Add the modules ID to the form html ID

Pull Request for Issue # .

### Summary of Changes
Adds the module ID to the HTML ID so that the HTML ID is always unique


### Testing Instructions
Add a login module to the front end
Inspect the source code and see that the login module has an id of id="login-form"
Install the patch
Inspect the login form and see that the module now has an ID with a module number added to the end of it.  e.g. id="login-form-16"

### Expected result
ID's should always be unique

### Actual result
ID is static but module can be replicated.
You see the correct module ID attached to the ID making it unique


### Documentation Changes Required
n/a
